### PR TITLE
I done diddley - about-page

### PR DESCRIPTION
**Pull request by CosmicStarfire**
did some things 
- sizing, adding link, need to possibly change button pop up media query break point.

### CODE REVIEW

File: css/about.css [hunk 1]
```diff
@@ -128,11 +126,12 @@ Media Queries (overriding styles for different screen sizes)
...
+		background: linear-gradient(
+			107deg,
+			rgba(9, 9, 9, 0.8) 9.51%,
+			rgba(9, 9, 9, 0) 76.72%
+		);
 		z-index: 1;
-
 	}
 
 	.hero-text {
```
Comment (maxharrisnet) [inline css/about.css]: The text alignment in the hero  doesn't seem to be working on desktop

Screen Shot 2024-01-30 at 11 37 52 AM

**Review by maxharrisnet — CHANGES_REQUESTED** [review]

File: about.html [hunk 2]
```diff
@@ -62,320 +121,292 @@
...
+			Through these experiences, I realize how the past, present, and future
+			are part of an overlapping continuum of human experience - like deep
+			roots coming out of a mountain.
+		</p>
+		<span class="signature">ITLCT Co-Founder, Natasha Rabsatt</span>
+		<div class="btn-container">
+			<a class="button3" href="/impact.html">Our Impact</a>
+		</div>
+	</section>
+	<!-- Footer -->
```
Comment (maxharrisnet) [inline about.html:197]: Please update the footer. The good version is in index.html in the development branch.

**Review by maxharrisnet — CHANGES_REQUESTED** [review]

File: css/about.css (hunk 1 above)
Comment (charlisable) [reply css/about.css]: text has been fixed

File: about.html (hunk 2 above)
Comment (charlisable) [reply about.html:197]: Added in updated footer

**Comment by charlisable** [comment]
Fixed the issues :)

**Review by maxharrisnet — APPROVED** [review]
Looks great, thanks for making the fixes!

**Review by SolivanLau — APPROVED** [review]
reviewed with group! looks great!

**Pull request merged**